client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -53,61 +87,198 @@ function HelpModal({
-                target='_blank'
-              >
-                placeholder
-              </a>
-              placeholder
-            </Trans>
-          </p>
-        </div>
-        <Button
-          block={true}
-          size='large'
-          variant='primary'
-          onClick={createQuestion}
-        >
-          {t('buttons.create-post')}
-        </Button>
-        <Spacer size='xxSmall' />
-        <Button
-          block={true}
-          size='large'
-          variant='primary'
-          onClick={closeHelpModal}
-        >
-          {t('buttons.cancel')}
-        </Button>
+                t={t}
+              />
+              <label htmlFor='description'>
+                {t('forum-help.whats-happening')}
+              </label>
+              <FormControl
+                id='help-modal-form-description'
+                name='description'
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                  setDescription(event.target.value);
+                }}
+                componentClass='textarea'
+                required={true}
+                style={{
+                  height: 100
+                }}
+                value={description}
+                maxLength={DESCRIPTION_MAX}
+                type='text'
+              />
+              {description.length < DESCRIPTION_MIN ? (
+                <p className='chars-left'>
+                  {t('learn.minimum-characters').replace(
+                    '{{characters}}',
+                    (DESCRIPTION_MIN - description.length).toString()
+                  )}
+                </p>
+              ) : (
+                <p className='chars-left'>
+                  {t('learn.characters-left').replace(
+                    '{{characters}}',
+                    (DESCRIPTION_MAX - description.length).toString()
+                  )}
+                </p>
+              )}
+            </div>
+            <Spacer size='xxSmall' />
+
+            <Button
+              block={true}
+              size='large'
+              variant='primary'
+              type='submit'
+              disabled={!canSubmitForm()}
+            >
+              {t('buttons.submit')}
+            </Button>
+            <Spacer size='xxSmall' />
+            <Button
+              block={true}
+              size='large'
+              variant='primary'
+              onClick={() => {
+                setShowHelpForm(false);
+              }}
+            >
+              {t('buttons.back')}
+            </Button>
+          </form>
+        ) : (
+          <>
+            <h3 className='help-modal-heading'>
+              <Trans i18nKey='learn.tried-rsa'>
+                <a
+                  href={RSA}
+                  rel='noopener noreferrer'
+                  target='_blank'
+                  title={t('learn.rsa')}
+                >
+                  placeholder
+                </a>
+              </Trans>
+            </h3>
```

Comment:
```suggestion
            <p className='help-modal-heading'>
              <Trans i18nKey='learn.tried-rsa'>
                <a
                  href={RSA}
                  rel='noopener noreferrer'
                  target='_blank'
                  title={t('learn.rsa')}
                >
                  placeholder
                </a>
              </Trans>
            </p>
```

This sentence is not acting as a heading, it is just regular content.